about.html

```diff
@@ -0,0 +1,137 @@
+            </li>
+            <li>
+              <a href="#">Logout</a>
+            </li>
+            <li>
+              <a href="#">Mypage</a>
+            </li>
+            <li>
+              <a href="#">English</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="navbar-main">
+        <div class="flex-container">
+          <button class="navbar-main__toggler" type="button">
+            <i class="fas fa-bars"></i>
+          </button>
+          <div class="navbar-main__nav">
+            <button class="navbar-button__close">x</button>
+            <a class="navbar-main__brand" href="index.html"
+              ><img
+                src="assets/images/air-sports-logo.jpg"
+                alt="Air Sports Logo"
+            /></a>
+            <ul class="navbar-main__list">
+              <li><a class="navbar-main__link" href="index.html">Home</a></li>
+              <li>
+                <a class="navbar-main__link" href="#">Safety Rules</a>
+              </li>
+              <li>
+                <a class="navbar-main__link" href="#features">Activities</a>
+              </li>
+              <li>
+                <a class="navbar-main__link" href="#staff">Our Instructors</a>
+              </li>
+              <li><a class="navbar-main__link" href="#">FAQ</a></li>
+            </ul>
+            <button id="cc-campaign" type="button">Book Now!</button>
+            <div class="menu-end">
+              <hr />
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section id="about-hero">
+      <div class="flex-container">
+        <h2>"Hello Aspiring Thrillers!"</h2>
+        <h1>About Air Sports Club</h1>
+        <div class="about-hero__message">
+          <p>
+            ​At Air Sports Club, we specialize in small group for multiple air
+            sport adventures. Over the past 28 years, Air Sports Club has grown
+            rapidly to become one of the leading sport club like adventure
+            companies. We Provide Instructors to help you do the air actively
+            and explore the adventure of Skydiving, Paragliding,
+            Paramotoring,Hang Gliding, Wingsuit Flying and more. We always dig
+            deeper in our new destinations to keep evolving our new locations or
+            spot with the core intention of keeping your experiences truly
+            original and the memories priceless. When you are doing this
+            activities, you just feel more Thrilled. When you see more
+            activities, you meet more people, create more lasting memories...
+            When you trust in our expertise to deliver you these extraordinary
+            experiences, it truly means the world to us.
+          </p>
+        </div>
+        <h4 class="about-hero__contact"></h4>
+          Please contact us on the email below for any further queries <br />
+          about our services.
+        </h4>
```

Comment:
Kindly make this centered in the page 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190355376-6610a9a6-8695-479c-80b8-ae30d0dd35f9.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190355492-d131e379-e071-4075-baf9-9f32deac3996.png)